src/components/Header/Header.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+//style
+import "./Header.sass";
+
+//components
+import { Typography } from "@material-ui/core";
+
+import { SearchBar } from "../SearchBar/SearchBar";
```

Comment:
Es preferible usar _rutas absolutas_ en los imports de componentes

```suggestion
import { SearchBar } from "components/SearchBar/SearchBar";
```